### ISSUE TITLE
FS with callbacks

### ISSUE THREAD

**Issue by PartiallyUntyped** [awslabs/llrt#38]
Since all fs methods are async in LLRT implementing callbacks for fs is trivial. They could be wrapped in a "callbackify" utility that will await the future and call a function with cb(err,result) rather than awaiting the result.

Size

S

**Comment by ajihyf** [comment]
@richarddavison I see that the issue was closed as “completed”, but there is no MR linked. Does this mean it is actually “not planned”?

**Comment by richarddavison** [comment]
@ajihyf you're right. It is not currently planned. If callback support is needed it can be provided via stubs. It's not super complex to add this but creates a lot of wrapper code that will increase the bin size. If you want, you can give this a shit and provide som stats around bin-size before and after this feature, if it's worth the extra size and complexity :)
